serialization/writer.py

```diff
@@ -0,0 +1,72 @@
+import font_pb2
+import pickle
+import sys
+
+# sys argv 1, ttx file name
+
+CHARS= ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L','M', 'N','O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X','Y', 'Z', 'a', 'b', 'c','d', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l','m', 'n', 'o', 'p', 'q','r', 's', 't', 'u', 'v', 'w', 'x','y', 'z', 'zero', 'one', 'two','three', 'four', 'five','six', 'seven', 'eight','nine', 'exclam','numbersign','dollar', 'percent','ampersand','asterisk','question','at']
+
+def pkl2protos(proto_dir, pickleName):
+    '''
+    This function takes one pickle and proto directory and puts all of its glyphs
+    into protos in the proto dir
+    
+    Paramaters
+    ----------
+
+    proto_dir: the directory where all the protos should be saved
+
+    pickle: the location of the pickle to be put into protos
+    '''
+    proto_dict = {glyph:eval("font_pb2.{}()".format(glyph)) for glyph in CHARS}
+    try:
+        with open(pickleName, 'rb') as fontString: 
+            fontDict = pickle.load(fontString)
+            for glyph in fontDict: 
+                # basically we are going to flatten everything into just an array of
+                # points
+                # we will keep track of the location of where each contour stops so we
+                # can reconstruct on the other end.
+                if glyph in proto_dict:
+                    contours = fontDict[glyph]
+                    num_contours = len(contours) 
+                    points = []
+                    contour_locations = []
+                    for c in contours:
+                        c = contours[c]
+                        contour_locations.append(len(c))
+                        for curve in c: 
+                            for point in curve: 
+                                points = [*points, point[0], point[1]]
+                    # now write it into the respective protobuf
+                    cur_proto = proto_dict[glyph]
+                    try: 
+                        with open(glyph, 'rb') as f: 
+                           cur_proto.ParseFromString(f.read())
+                    except: 
+                        pass
```

Comment:
What would cause this to raise an exception? Do we want to log this?

Reply:
I think it would trigger if the file is in the wrong format. I will add some logging.

Reply:
Actually that is not 100% accurate. The first time around the protobuf doesnt exist, and will cause an error. This try catch is so that it wont cause an error the first time around